ActionSheet.android.js

```diff
@@ -161,8 +159,7 @@ export default class ActionSheet extends React.Component {
     BackAndroid.addEventListener('actionSheetHardwareBackPress', this._animateOut);
   }
 
-  @autobind
-  _onSelect(index) {
+  _onSelect = (index) => {
```

Comment:
I prefer bind since it's less likely to change in the future. The binding semantics of class properties could change in a future version of Babel/JS and we don't want to deal with breakage.